backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/TopDownAnalyzerFacadeForKonan.kt

```diff
@@ -56,8 +57,12 @@ object TopDownAnalyzerFacadeForKonan {
             config: KonanConfig
     ): AnalysisResult {
 
-        // we print out each file we compile for now
-        files.forEach{println(it)}
+        // we print out each file we compile if frontend phase is verbose
```

Comment:
lower1 will not be verbose in this case. get() returns a list of phase names marked as verbose